extension/ext-comments.css

```diff
@@ -273,18 +272,18 @@ a:focus {
 
 /* Why does substack remove focus indicators from everything?! */
 .ext-comments .comment-editor textarea:focus {
-  border: 2px solid var(--background_pop) !important;
```

Comment:
This looks kinda ugly to me, since with the option off, textboxes now have both a black border and a blue outline.

Maybe you can toggle a class on the document <body> element when the classic style option is active?

Then we can distinguish styles like:

    body .ext-comment > .. { something }
    body.classic-style .ext-comment > .. { something else }